### PR TITLE
test: Implement extension installation and basic extension handling test scenario

### PR DESCRIPTION
**Pull request by amisskii**
### What does this PR do?

This PR adds a test scenario for the minikube extension

### What issues does this PR fix or reference?

https://github.com/containers/podman-desktop-extension-minikube/issues/137

### How to test this PR?

- Clone the podman-desktop repo
- cd to podman-desktop folder
- Execute pnpm install
- Execute pnpm build
- Set environment variable executing: export PODMAN_DESKTOP_ARGS="/path/to/podman-desktop"
- Clone the podman-desktop-extension-minikube repo
- cd  to  podman-desktop-extension-minikube folder
- Execute pnpm install
- Execute pnpm test:e2e

### CODE REVIEW

**Comment by ScrewTSW** [comment]
A lot of your files are missing an empty line at the end of the file :D also, prcheck is failing

File: tests/playwright/src/spec/minikube-extension.spec.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import {
```
Comment (odockal) [inline tests/playwright/src/spec/minikube-extension.spec.ts:1]: Missing copyright

File: vitest.config.js [hunk 2]
```diff
@@ -18,8 +18,16 @@
 
 import path from 'node:path';
 
+const excludeArray = [
```
Comment (odockal) [inline vitest.config.js:21]: copy right year 2023-2024

File: tests/playwright/src/spec/minikube-extension.spec.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    waitForPodmanMachineStartup, 
+} from '@podman-desktop/tests-playwright';
+
+const EXTENSION_IMAGE: string = 'ghcr.io/podman-desktop/podman-desktop-extension-minikube:nightly';
+const EXTENSION_NAME: string = 'minikube';
+const EXTENSION_LABEL: string = 'podman-desktop.minikube';
+
+let extensionsPage: ExtensionsPage; 
+
+const skipExtensionInstallation = process.env.SKIP_EXTENSION_INSTALL ? process.env.SKIP_EXTENSION_INSTALL : false;
```
Comment (odockal) [inline tests/playwright/src/spec/minikube-extension.spec.ts:14]: `= process.env.SKIP_EXTENSION_INSTALL === 'true'` should be sufficient and correct.

File: tests/playwright/src/spec/minikube-extension.spec.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+
+test.beforeAll(async ({ runner, page, welcomePage }) => {
+    runner.setVideoAndTraceName('minikube-extension-e2e');
+    await welcomePage.handleWelcomePage(true);
+    await waitForPodmanMachineStartup(page);
```
Comment (odockal) [inline tests/playwright/src/spec/minikube-extension.spec.ts:19]: do we need podman machine at this point?

**Review by odockal — CHANGES_REQUESTED** [review]

**Comment by odockal** [comment]
Otherwise it is good. I have tested it locally.

File: tests/playwright/src/spec/minikube-extension.spec.ts (hunk 1 above)
Comment (odockal) [reply tests/playwright/src/spec/minikube-extension.spec.ts:19]: please just 2024.

**Review by odockal — APPROVED** [review]
Only one copyright update, otherwise LGTM.

**Pull request merged**